sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewSubscriber.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.servicebus.implementation.LockContainer;
+import org.reactivestreams.Subscription;
+import reactor.core.CoreSubscriber;
+import reactor.core.publisher.BaseSubscriber;
+import reactor.core.publisher.Mono;
+import reactor.util.context.Context;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import java.util.function.Function;
+
+/**
+ * Receives messages from to upstream, pushes them downstream and start lock renewal.
+ */
+final class LockRenewSubscriber extends BaseSubscriber<ServiceBusReceivedMessage> {
+    private static final OffsetDateTime COMPLETED_RENEW_TIME = OffsetDateTime.now();
```

Comment:
Why OffsetDateTime.now()? This variable is only used once. 
I'm not sure why we don't just have a LockRenewalOperation.DONE it just returns empty.

```java
class LockRenewalOperation {
    static final LockRenewalOperation COMPLETED = new LockRenewalOperation(null, null, null, null);
}
```

Reply:
Removed it totally based on other comments.